Packet++/header/SSLLayer.h

```diff
@@ -281,28 +299,29 @@ namespace pcpp
 	                                |                                yyy
 	 @endverbatim
 
-	 * Second example is a multiple-message handshake layer comprises of server-hello, certificate and server-key-exchange
-	 * messages:
+	 * Second example is a multiple-message handshake layer comprises of server-hello, certificate and
+	 * server-key-exchange messages:
 
 	 @verbatim
 
-	         |---------------------------------------------- SSLHandshakeLayer -----------------------------------------------------|
+	         |---------------------------------------------- SSLHandshakeLayer
+	 -----------------------------------------------------|
 	         +----------------------+-------------------------------------+---------------------------+-----------------------------+
-	         | ssl_tls_record_layer |       SSLServerHelloMessage         |   SSLCertificateMessage   | SSLServerKeyExchangeMessage |
-	         |        struct        |                                     |                           |                             |
+	         | ssl_tls_record_layer |       SSLServerHelloMessage         |   SSLCertificateMessage   |
+	 SSLServerKeyExchangeMessage | |        struct        |                                     | | |
 	         +----------------------+-------------------------------------+---------------------------+-----------------------------+
-	          /     |       \               |          \         \               |           \               |            \
-	         /    version    \      |   handshake       \        rest of  |      |          rest      |      |            rest
-	        /     TLS1_0      \            type          \       message      handshake   of fields...   handshake    of fields...
+	          /     |       \               |          \         \               |           \               | \ /
+	 version    \      |   handshake       \        rest of  |      |          rest      |      |            rest /
+	 TLS1_0      \            type          \       message      handshake   of fields...   handshake    of fields...
 	     type                  \    | SSL_SERVER_HELLO    \      fields...|     type                  |     type
-	 SSL_HANDSHAKE           length                   handshake             SSL_CERTIFICATE             SSL_SERVER_KEY_EXCHANGE
-	     (22)                 xxx   |               version,length        |                           |
+	 SSL_HANDSHAKE           length                   handshake             SSL_CERTIFICATE SSL_SERVER_KEY_EXCHANGE (22)
+	 xxx   |               version,length        |                           |
```

Comment:
clang-format completely messed it up 😕

Reply:
ops, I noticed the above one but missed the below one. fixed.